docs/GettingStarted.md

```diff
@@ -0,0 +1,106 @@
+    - [Installing the front-end bundle](#installing-the-front-end-bundle)
+    - [Going further](#going-further)
+
+<p align="center">
+  <a href="https://crawler.algolia.com/admin/netlify"><img width="223" alt="Sign in to Algolia with Netlify" src="https://user-images.githubusercontent.com/1637651/95232629-0ebc1d00-0805-11eb-9b77-f116a3ed1a2b.png">
+</a>
+
+## Getting started
+
+Here's how you can implement an instant search on your site using Netlify, Algolia, and this plugin. **All you need is an existing Netlify site.**
+
+> This plugin is in Public Beta
+
+### Link your site to Algolia
+
+First, go to the [Algolia Crawler for Netlify](https://crawler.algolia.com/admin/netlify) and click **Sign in to Algolia with Netlify**. We automatically create a new Algolia account if you don't have one.
+
+<img src="/docs/screenshots/signin.png?raw=true" alt="Sign in to Algolia with Netlify">
+
+Then, authorize Algolia to access your Netlify account. The plugin only updates your plugin settings and adds necessary environment variables.
+
+<img src="/docs/screenshots/authorize.png?raw=true" alt="Authorize Algolia's OAuth">
+
+In the [Crawler Admin Console](https://crawler.algolia.com/admin/netlify), search for your site, then click **Install**. We automatically update your Netlify site to add the environment variables (prefixed by `ALGOLIA_`) that you need to use the plugin.
+
+We also create an Algolia application with a dedicated [Free plan](https://www.algolia.com/pricing/).
+
+<img src="/docs/screenshots/search-site.png?raw=true" alt="Search for your site">
+
+<img src="/docs/screenshots/accept-install.png?raw=true" alt="Install in your site">
+
+The plugin is now correctly installed and ready to index your site.
+
+<img src="/docs/screenshots/installed.png?raw=true" alt="View your site information.">
+
+### Indexing
+
+Once you've installed the plugin, your next Netlify deploy will trigger a crawl and populate an Algolia index. You can manually trigger a new deploy in Netlify by clicking **Retry deploy > Deploy site** on any deploy.
+
+When it receives a build hook, the Algolia Crawler processes your website asynchronously. This operation takes some time, resulting in a short delay between the first deploy and the associated crawl. Your site and your Algolia index will be out of sync during that delay.
+
+You can find information about your current crawler in the [Netlify deploy logs](https://docs.netlify.com/monitor-sites/logs/#deploy-log).
+
+> Note: by default, we only build the `master` branch. We can however create one crawler (targeting one Algolia index) per Git branch, so that you can have, for example, a production index on `master` and development index on `develop`. You need to configure the [`branches` plugin input](../plugin#available-parameters) to enable this feature.
+> If you're using our front-end library, you'll also need to [pass the correct branch](../frontend#using-multiple-branches) to the library parameters.
```

Comment:
Still not good

Reply:
works for me, what does not work for you?

Reply:
Indeed it's the one below sorry: `Using Multiple branches`

Reply:
arg yes